.github/workflows/update-currencies.yml

```
@@ -2,99 +2,53 @@
-          # Create a pull request
-          curl -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
-               -X POST \
-               -d '{"title": "Update currencies", "head": "update-currencies", "base": "main"}' \
-               https://api.github.com/repos/${{ github.repository }}/pulls
-
-      - name: Notify no changes
-        if: env.changes == 'false'
-        run: echo "Currencies up-to-date with API, skipping commit."
+      - name: Create Pull Request
+        if: env.changed == 'true'
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)